test/integration/test_payroll_process_scenarios_3/dataEmployees.js

```diff
@@ -0,0 +1,581 @@
+    display_name : 'ANTOINE LEMOINE DUFRESNE',
+    sex : 'M',
+    payment_uuid : null,
+    payroll_configuration_id : '24',
+    currency_id : '2',
+    payment_date : null,
+    base_taxable : 0,
+    basic_salary : 0,
+    gross_salary : 0,
+    grade_salary : 300,
+    text : 'Des hôpitaux adjoints',
+    net_salary : 0,
+    working_day : 0,
+    total_day : 0,
+    daily_salary : 0,
+    amount_paid : 0,
+    status_id : 1,
+    status : 'PAYROLL_STATUS.WAITING_FOR_CONFIGURATION',
+    balance : 0,
+    hrreference : 'EM.TE.1161',
+    cost_center_id : 6,
+    service_name : 'Surgery',
+    negativeValue : 0,
+  },
+  {
+    employee_uuid : '439A7E097A8E4B5AAB7E165C870FDA8E',
+    reference : 1034,
+    code : 'EMP-STAFF-01025',
+    hiring_date : '1991-10-31',
+    nb_enfant : 0,
+    individual_salary : 0,
+    account_id : 366,
+    creditor_uuid : 'DA3EA4642F0C47BE99413C80433E78E7',
+    function_name : null,
+    display_name : 'ANTONELLA NAVARRO VILLALOBOS',
+    sex : 'F',
+    payment_uuid : null,
+    payroll_configuration_id : '24',
+    currency_id : '2',
+    payment_date : null,
+    base_taxable : 0,
+    basic_salary : 0,
+    gross_salary : 0,
+    grade_salary : 325,
+    text : 'Médecin chef clinique',
+    net_salary : 0,
+    working_day : 0,
+    total_day : 0,
+    daily_salary : 0,
+    amount_paid : 0,
+    status_id : 1,
+    status : 'PAYROLL_STATUS.WAITING_FOR_CONFIGURATION',
+    balance : 0,
+    hrreference : 'EM.TE.1034',
+    cost_center_id : 3,
+    service_name : 'Internal Medicine',
+    negativeValue : 0,
+  },
+  {
+    employee_uuid : '1D7B1BC4E5264C8D8A87145962147131',
+    reference : 1203,
+    code : 'EMP-STAFF-01194',
+    hiring_date : '2012-07-31',
+    nb_enfant : 0,
+    individual_salary : 0,
+    account_id : 366,
+    creditor_uuid : '8AB76E1687824337BABDB48DB87C7954',
+    function_name : null,
+    display_name : 'ANTONIO CRUZ MENDEZ',
+    sex : 'M',
+    payment_uuid : null,
+    payroll_configuration_id : '24',
+    currency_id : '2',
+    payment_date : null,
+    base_taxable : 0,
+    basic_salary : 0,
+    gross_salary : 0,
+    grade_salary : 325,
+    text : 'Médecin chef clinique',
+    net_salary : 0,
+    working_day : 0,
+    total_day : 0,
+    daily_salary : 0,
+    amount_paid : 0,
+    status_id : 1,
+    status : 'PAYROLL_STATUS.WAITING_FOR_CONFIGURATION',
+    balance : 0,
+    hrreference : 'EM.TE.1203',
+    cost_center_id : 2,
+    service_name : 'Billing and Accounting Department',
+    negativeValue : 0,
+  },
+  {
+    employee_uuid : '4D51E9F3FEB24F1693DA4F0F90DED8C7',
+    reference : 1141,
+    code : 'EMP-STAFF-01132',
+    hiring_date : '2009-10-31',
+    nb_enfant : 0,
+    individual_salary : 0,
+    account_id : 366,
+    creditor_uuid : 'DA14C0A873A04E6C8EEAF749A099E363',
+    function_name : null,
+    display_name : 'ANTONIO DE SANTIS FERRARI',
+    sex : 'M',
+    payment_uuid : null,
+    payroll_configuration_id : '24',
+    currency_id : '2',
```

Comment:
Should these be integers instead of strings?  Things like `currency_id`, `payroll_configuration_id`, etc?